test/set-version-test.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
```

Comment:
What are you testing in this file? I have to read further down to understand you're comparing two files with each other. What is being compared exactly? And is testing if two files are identical the best approach? Maybe it is, but it seems a bit superfluous. And if something else changes you may have to rewrite your tests each time.